src/applications/vaos/appointment-list/components/AppointmentsPage/AppointmentsPageV2.jsx

```diff
@@ -0,0 +1,279 @@
+      //   selectedDateRange.startDate,
+      //   selectedDateRange.endDate,
+      //   0,
+      // );
+      history.push('/past');
+    } else if (e.currentTarget.value === 'cancelled') {
+      history.push('/cancelled');
+    }
+  }
+
+  return (
+    <PageLayout>
+      <h1 className="vads-u-flex--1">{pageTitle}</h1>
+      <DowntimeNotification
+        appTitle="VA online scheduling tool"
+        isReady
+        dependencies={[externalServices.vaosWarning]}
+        render={(props, childContent) => (
+          <WarningNotification {...props}>{childContent}</WarningNotification>
+        )}
+      />
+
+      {showScheduleButton && (
+        <ScheduleNewAppointment
+          isCernerOnlyPatient={isCernerOnlyPatient}
+          showCommunityCare={showCommunityCare}
+          showDirectScheduling={showDirectScheduling}
+          showHomePageRefresh={showHomePageRefresh}
+          startNewAppointmentFlow={() => {
+            recordEvent({
+              event: `${GA_PREFIX}-schedule-appointment-button-clicked`,
+            });
+            startNewAppointmentFlow();
+          }}
+        />
+      )}
+
+      {showCheetahScheduleButton && (
+        <ScheduleNewProjectCheetah
+          showHomePageRefresh={showHomePageRefresh}
+          startNewAppointmentFlow={() => {
+            recordEvent({
+              event: `${GA_PREFIX}-schedule-project-cheetah-button-clicked`,
+            });
+            startNewAppointmentFlow();
+          }}
+        />
+      )}
+
+      {!expressCare.enabled && (
+        <>
+          {showPastAppointments && <TabNav />}
+          {routes}
+        </>
+      )}
+      {expressCare.enabled && (
+        <>
+          {isLoading && (
+            <LoadingIndicator message="Loading your appointment information" />
+          )}
+          {!isLoading && (
+            <>
+              {!isCernerOnlyPatient && (
+                <RequestExpressCare
+                  {...expressCare}
+                  showHomePageRefresh={showHomePageRefresh}
+                  startNewExpressCareFlow={() => {
+                    recordEvent({
+                      event: `${GA_PREFIX}-express-care-request-button-clicked`,
+                    });
+                    startNewExpressCareFlow();
+                  }}
+                />
+              )}
+              {expressCare.hasRequests && (
+                <h2 className="vads-u-font-size--h3 vads-u-margin-y--3">
+                  Your appointments
+                </h2>
+              )}
+
+              <label className="vads-u-display--inline-block vads-u-margin-top--0 vads-u-margin-right--2">
+                Show by type <span className="sr-only" />
+              </label>
+              <Select
+                options={options}
+                onChange={onChange}
+                aria-labelledby="options"
+              />
+              {routes}
+            </>
+          )}
+        </>
+      )}
+      <CancelAppointmentModal
```

Comment:
Can remove this, since we're not cancelling appointments on this page.

Reply:
Done.